ui/src/components/tag/TagList.js

```diff
@@ -235,13 +246,30 @@ export default class TagList extends React.Component {
         };
     }
 
+    groupMetaObject(group) {
+        return {
+            selfServe: group.selfServe,
+            reviewEnabled: group.reviewEnabled,
+            notifyRoles: group.notifyRoles,
+            serviceExpiryDays: group.serviceExpiryDays,
+            userAuthorityExpiration: group.userAuthorityExpiration,
+            userAuthorityFilter: group.userAuthorityFilter,
+            memberExpiryDays: group.memberExpiryDays,
+            tags: group.tags,
+        };
+    }
+
     updateMeta(meta, csrf, successMessage, showSuccess = true) {
+        var catagoryObject = this.props.domain;
+        if (this.state.category === 'role') {
+            catagoryObject = this.props.role;
+        } else if (this.state.category === 'group') {
+            catagoryObject = this.props.group;
+        }
         this.api
             .putMeta(
                 this.props.domain,
-                this.state.category === 'domain'
-                    ? this.props.domain
-                    : this.props.role,
+                catagoryObject,
                 meta,
                 'Updated ' + this.props.domain + ' Meta using Athenz UI',
```

Comment:
we should update this message to display the appropriate categoryObject now

Reply:
also while you are at it, please update the categoryObject spelling catagoryObject -> categoryObject

Reply:
Fixed